packages/@aws-cdk/aws-lambda/lib/function-base.ts

```diff
@@ -62,6 +62,11 @@ export interface IFunction extends IResource, ec2.IConnectable, iam.IGrantable {
    */
   readonly architecture: Architecture;
 
+  /**
+   * The ARN(s) to put into the resource field of the generated IAM policy for grantInvoke()
+   */
```

Comment:
Let's add some extra detail to this comment that users should not use this if they can avoid it, and should use `grantInvoke` instead